Ofast deprecation clarifications (#101005)
llvm/llvm-project: clang/docs/ReleaseNotes.rst

Deprecated Compiler Flags
-------------------------

> - The ``-Ofast`` command-line option has been deprecated, but there is no
>   timeline for removal yet. Thus, the main effect of emitting a deprecation
>   warning message is to discourage its usage due to the problems of ``-Ofast``:
>   it enables both the ``-O3`` optimization-level as well as non-standard
>   ``-ffast-math`` behaviors and as such it is perceived to be misleading as an
>   optimization level.  Users are advised to switch to ``-O3 -ffast-math`` if

Reviewer: ```suggestion
- The ``-Ofast`` command-line option has been deprecated, but there is no
  timeline for removal. Deprecation is intended to discourage use of the flag
  due to the problems of ``-Ofast``: it enables both the ``-O3`` optimization
  level as well as non-standard ``-ffast-math`` behaviors and as such it is
  misleading as an optimization level.  Users are advised to switch to
  ``-O3 -ffast-math`` if
```
